backend/PennCourses/docs_settings.py

```diff
@@ -1060,6 +1060,34 @@ def get_description(self, path, method):
             desc += '\n\n<span style="color:red;">User authentication required</span>.'
         return desc
 
+    # Overrides, uses overridden method
+    def map_serializer(self, serializer):
+        """
+        This method adds property docstrings as field descriptions when appropriate, in addition
+        to calling the overridden map_serializer function.
```

Comment:
Added a link to the docs -- I think adding a link to the line in the GitHub code would be bad practice since it will become incorrect if any changes are made to that document